### PR TITLE
Update Tailwind example to use Expo Router

### PR DESCRIPTION
**Pull request by SeanBarker182**
This PR was created to resolve #287 

It accomplishes the following:

- Updates Expo to v49 along with its dependencies
- Updates Solito to use v4
- Switches the router to use Expo Router v2
- Updates README & Docs

@nandorojo Would you prefer for me to update the dependencies on the other examples as well to keep things consistent? I can also switch the Next app to use the app directory if preferred.

### CODE REVIEW

**Comment by nandorojo** [comment]
thanks for this! would be great to switch the rest of the examples too. let's not move to app dir yet though

**Comment by nandorojo** [comment]
awesome, thank you! given your testing, i’ll be merging this in somewhat blindly, so i just want to be sure it’s all working properly

**Comment by nandorojo** [comment]
i assume you used `expo upgrade` everywhere, right?

File: example-monorepos/with-tailwind/readme.md [hunk 1]
```diff
@@ -87,22 +87,22 @@ Take a look at the [`packages/app/design`](https://github.com/nandorojo/solito/t
...
 - `apps` entry points for each app
 
-  - `expo`
+   - `expo`
+    - `app` you'll be creating files inside of `apps/expo/app` to use file system routing on iOS and Android.
   - `next`
 
 - `packages` shared packages across apps
   - `app` you'll be importing most files from `app/`
     - `features` (don't use a `screens` folder. organize by feature.)
```
[the remark this answers is not in the document]
Comment (SeanBarker182) [reply example-monorepos/with-tailwind/readme.md:104]: I don't really understand what it's trying to communicate with those highlights to be honest. Just double checked and they're not removed.

**Comment by SeanBarker182** [comment]
> i assume you used `expo upgrade` everywhere, right?

Correct!

**Pull request merged**